GOVERNANCE.md

```diff
@@ -72,6 +72,14 @@ Functionality that is out of current scope:
 * How to obtain this role: (1) merge into master a PR adding role="cre" to DESCRIPTION, and (2) submit updated package to CRAN (previous CRAN Maintainer will have to confirm change by email to CRAN). 
 * How this role is recognized: credited via role="cre" in DESCRIPTION, so they appear as Maintainer on CRAN. 
 
+## Community Engagement Coordinator
+
+* Definition: An individual who is involved in the project but does **not** also occupy the Committer or CRAN Maintainer role.  In charge of maintaining The Raft blog, preparing Seal of Approval Applications, addressing Code of Conduct violations, and planning social or community events.
+
+* How to obtain this role: Contact the current Community Engagement Coordinator.
```

Comment:
I think more important for this role is "how a new CEC is selected", it might make sense to think of how you'd pass on responsibilities if you so chose in the future. Would it just be someone chosen by you? Should we get Consensus from existing Committers?